buildlib/jucx/jucx-publish.yml

```diff
@@ -72,9 +95,13 @@ jobs:
           # Maven requires version to be of form MAJOR_VERSION.MINOR_VERSIOn,...
           # ucx tags are of form v1.x.x - need to remove 'v' from the beginning of string
           MAVEN_VERSION=${TAG:1}
-          make -C bindings/java/src/main/native/ ${{ parameters.target }} \
+          make -C bindings/java/src/main/native/ publish-release \
               ARGS="--settings ${{ parameters.temp_cfg }}" JUCX_VERSION=${MAVEN_VERSION}
+          set -x
+          jar=$(find . -name 'jucx-*[0-9].jar')
+          jar tf "${jar}"
         displayName: Publish JUCX jar to maven central
+        condition: eq(variables['Build.Reason'], 'IndividualCI')
```

Comment:
are the steps "Generate temporary config" and "DownloadSecureFile@1" also needed only if `eq(variables['Build.Reason'], 'IndividualCI')` ?

Reply:
Correct, both the config and secure files are required for Maven publish.
Added condition to skip the config step. Secure file download is skipped by default.